src/main/java/domain/PrizeCondition.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+import static domain.Prize.*;
+
+public enum PrizeCondition {
+    FIRST(FIRST_PRIZE_MATCHING_NUMBER, FIRST_PRIZE_REWARD),
+    THIRD(THIRD_PRIZE_MATCHING_NUMBER, THIRD_PRIZE_REWARD),
+    FOURTH(FOURTH_PRIZE_MATCHING_NUMBER, FOURTH_PRIZE_REWARD),
+    FIFTH(FIFTH_PRIZE_MATCHING_NUMBER, FIFTH_PRIZE_REWARD);
```

Comment:
상수를 따로 분리해주시는 이유가 있을까요? 😄 
```suggestion
    FIRST(6, 2_000_000_000),
    THIRD(5, 1_500_000),
    FOURTH(4, 50_000),
    FIFTH(3, 5_000);
```

Reply:
이전에 enum을 만들기 전에 작성한 상수값인데, enum 작성 이후에는 too much인 느낌이 드네요 👍